lib/internal/test_runner/coverage.js

```diff
@@ -51,13 +51,19 @@ class CoverageLine {
 }
 
 class TestCoverage {
-  constructor(coverageDirectory, originalCoverageDirectory, workingDirectory, excludeGlobs, includeGlobs, thresholds) {
```

Comment:
**Review note** This was changed to accept an `options` because of the increasing size of arguments for this constructor.

Reply:
If you're going to introduce an `options` object, we might as well pass all of the arguments in it.